manifests/repo/apt_postgresql_org.pp

```diff
@@ -5,12 +5,18 @@
   #
   # http://www.postgresql.org/download/linux/debian/
   #
+  $default_baseurl = 'http://apt.postgresql.org/pub/repos/apt/'
+  $baseurl_real = pick($postgresql::repo::baseurl,$default_baseurl,'undefined')
+  if ( $baseurl_real == 'undefined' ){
+    fail('No repo baseurl defined or automatically detected.')
```

Comment:
I'm guessing this condition, would really only occur if there was a bug in the pick code. I'm just curious if this check is even necessary, the pick chooses a variable you set directly above it ($default_baseurl) as a backup to the user provided one ... I'm just pondering if this is even just overkill doing an undefined match.

Bugs in the code can be (probably already are) testing with rspec-puppet, at least if we test for a yumrepo resource, with an expected repo url ... we're done here, no runtime test needed.